libs/evcs-service/src/test/java/uk/gov/di/ipv/core/library/client/EvcsClientTest.java

```diff
@@ -191,7 +191,7 @@ void testGetUserVCs_emptyListIsReturned() throws Exception {
     }
 
     @ParameterizedTest
-    @ValueSource(ints = {150, 400})
+    @ValueSource(ints = {400, 500})
```

Comment:
IMO we shouldn't worry about 1XX status codes - they're really internal HTTP processing codes that we won't see